app/src/main/java/com/codesoom/assignment/controller/ProductController.java

```diff
@@ -0,0 +1,102 @@
+package com.codesoom.assignment.controller;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.dto.ProductDto;
+import com.codesoom.assignment.exception.ProductNotFoundException;
+import com.codesoom.assignment.service.ProductService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+//TODO        고양이 장난감 목록 얻기 - GET /products - 테스트 미작성, 장난감 등록 후 확인
+//TODO        고양이 장난감 상세 조회하기 - GET /products/{id}  - 테스트 미작성, 장난감 등록 후 확인
+//TODO        고양이 장난감 등록하기 - POST /products -테스트미작성, RequestBody Validation 추가해야하는가?(항상 파라미터가 다 들어온다고 가정해야 하는가?)
+//TODO        고양이 장난감 수정하기 - PATCH /products/{id}   -테스트 미작성
+//TODO        고양이 장난감 삭제하기 - DELETE /products/{id}  -테스트 미작성
+
+/**
+ * 장난감 CURD 컨트롤러.
```

Comment:
다음은 제가 방금 작성한 이 클래스의 용도를 설명하는 문장입니다. 두 문장의 차이점과 장단점이 어떻게 될까요?

```suggestion
 * 상품 정보와 관련된 HTTP 요청을 처리합니다.
```